### ISSUE TITLE
Crahses on Pi 3

### ISSUE THREAD

**Issue by Niall7459** [ChristopheJacquet/PiFmRds#97]
It keeps making my raspberry pi hang!

**Comment by MinePro120** [comment]
It does that to my pi too. When I try minimizing a window, the pi freezes. As I wanted a dedicated radio station, I bought another sd, made the command run at startup and change to CLI instead of GUI Desktop. It worked perfect.

**Comment by Niall7459** [comment]
Yes cli seems to fix this issue. And it’s caused by pi fm adv using the same DMA engine as the graphics buffer this is fixed in a fork by @SaucySoliton
